awscli/examples/global_options.rst

```diff
@@ -70,3 +70,7 @@
   
   The maximum socket connect time in seconds. If the value is set to 0, the socket connect will be blocking and not timeout. The default value is 60 seconds.
   
+``--migrate-v2`` (boolean)
+  
+  Enable AWS CLI v2 migration assistance. Prints warnings if the command would face a breaking change after swapping AWS CLI v1 for AWS CLI v2 in the current environment. Prints one warning for each breaking change detected.
+  
```

Comment:
Thinking about it during review, `--migrate-v2` feels a little too "action-y" like we're performing some sort of migration. What do folks think about `--v2-debug` or `--migration-debug`?